test_accounts/verification_test_accounts.md

```diff
@@ -2,36 +2,48 @@
-  3. **Don’t change any preselected answers** when asked about receiving an authentication code. Just click “Continue” to go to the next step.
-
-**Do not modify any of the attributes for these test accounts, including passwords**. If you need to use one of these accounts for a different use case, [contact us](https://developer.va.gov/support/contact-us) on our developer portal.
-
-**Password for all ID.me accounts: Password1234!**
-
-|Email|First Name|Middle Name|Last Name|Sex|Birthdate|SSN|Veteran Verification Status|Veteran Confirmation Status|Disability Rating|
-| :--- | :--- | :--- | :--- | :--- | :--- | :--- | :--- | :--- | :--- |
-|va.api.user+idme.001@gmail.com|Tamara|E|Ellis|F|1967-06-19|796130115|confirmed|confirmed|40|
-|va.api.user+idme.002@gmail.com|Janet|L|Moore|F|1949-05-06|796127677|confirmed|confirmed|50|
-|va.api.user+idme.003@gmail.com|Ralph|E|Lee|M|1948-10-30|796378782|confirmed|confirmed|60|
-|va.api.user+idme.005@gmail.com|Pauline|E|Foster|F|1976-06-09|796330625|confirmed|confirmed|60|
-|va.api.user+idme.006@gmail.com|Russell|James|Freeman|M|1969-11-05|796149080|confirmed|confirmed|30|
-|va.api.user+idme.008@gmail.com|Greg|A|Anderson|M|1933-04-05|796121200|not confirmed|not confirmed|50|
-|va.api.user+idme.012@gmail.com|Andrea|L|Mitchell|F|1959-12-01|796127781|confirmed|confirmed|30|
-|va.api.user+idme.013@gmail.com|Kenneth|William|Andrews|M|1990-02-20|796295980|confirmed|confirmed|40|
-|va.api.user+idme.025@gmail.com|Wesley|Watson|Ford|M|1986-05-06|796043735|confirmed|confirmed|100|
-|va.api.user+idme.026@gmail.com|Melvin|V|Freeman|M|1971-11-19|796184750|confirmed|confirmed|100|
-|va.api.user+idme.027@gmail.com|Herbert|Michael|Gardner|M|1983-02-21|796122369|confirmed|confirmed|40|
-|va.api.user+idme.029@gmail.com|Jesse|J|George|M|1950-01-31|796330163|confirmed|confirmed|30|
-|va.api.user+idme.030@gmail.com|Sidney|A|Gibson|M|1933-08-04|796127094|confirmed|confirmed|null|
-|va.api.user+idme.032@gmail.com|Jeffery|J|Hayes|M|1937-09-25|796131729|confirmed|confirmed|30|
-|va.api.user+idme.033@gmail.com|Everett|Avery|Horton|M|1982-04-23|796377148|confirmed|confirmed|null|
-|va.api.user+idme.034@gmail.com|Mathew|A|Howell|M|1927-01-18|796131275|confirmed|confirmed|null|
-|va.api.user+idme.035@gmail.com|Julio|E|Hunter|M|1951-11-18|796378321|confirmed|confirmed|30|
-|va.api.user+idme.037@gmail.com|Daryl|F|Lawrence|M|1953-02-15|796153447|confirmed|confirmed|40|
-|va.api.user+idme.041@gmail.com|Christian|Fitzgerald|Patterson|M|1964-03-04|796218467|confirmed|confirmed|10|
-|va.api.user+idme.042@gmail.com|Jessie|F|Price|M|1934-04-07|796126978|confirmed|confirmed|10|
-|va.api.user+idme.046@gmail.com|Mattie|May|Reid|F|1964-04-14|796109651|confirmed|confirmed|null|
+  2. Select **Login.gov** to sign in to the sandbox environment.
+  3. Enter the Login.gov username and password (**Password12345!!!**).
+  4. Use the Login.gov MFA seed to generate a 2FA code using an application like Google Authenticator or Authy.
```

Comment:
"like" should be "such as" in this context. Such as gives concrete examples of MFA apps that can be used, whereas like would exclude apps that could work for our user. I'm also going to suggest removing the two instances of the word "use."
```suggestion
  4. Use the Login.gov MFA seed to generate a 2FA code with an app such as Google Authenticator or Authy.
```